action.yml

```diff
@@ -0,0 +1,57 @@
+# azure spirng cloud action
+name: 'AzureSpringCloud'
+description: 'Deploy applications to Azure Spring Cloud and manage deployments.'
+inputs: 
+  azure-subscription:
+    description: 'Select the Azure Resource Manager subscription for the deployment.'
+    required: true
+  action:
+    description: 'Action to be performed on Azure Spring Cloud.'
+    required: true
+    default: 'Deploy'
+  service-name:
+    description: 'Select the Azure Spring Cloud service to which to deploy.'
+    required: true
+  app-name:
+    description: 'Select the Azure Spring Cloud app to deploy.'
+    required: true
+  use-staging-deployment:
+    description: "Automatically select the deployment that's set as Staging at the time the task runs."
+    required: true
+    default: true
+  create-new-deployment:
+    description: "Whether to target the deployment that's set as Staging at the time of execution. If unchecked, the 'Deployment Name' setting must be set."
```

Comment:
The comment does not match the argument name. Expected?

Reply:
Expected, copied form Azure DevOps pipeline tasks. If create-new-deployment is true, it means to create a new deployment called staging.